resources/js/store/modules/user.js

```diff
@@ -50,6 +50,37 @@ const actions = {
       return Promise.reject(error);
     }
   },
+  async getInfo({ commit }) {
+    try {
+      const {
+        access_token: accessToken,
```

Comment:
em nghĩ nên convert snakecase to camelcase ở trong base request luôn để khỏi mỗi lần gọi phải convert lại.